cmd/cloud-controller-manager/app/controllermanager.go

```diff
@@ -26,10 +26,12 @@ import (
 
 	"github.com/spf13/cobra"
 
+	utilruntime "k8s.io/apimachinery/pkg/util/runtime"
 	"k8s.io/apimachinery/pkg/util/sets"
 	"k8s.io/apimachinery/pkg/util/uuid"
 	"k8s.io/apimachinery/pkg/util/wait"
 	"k8s.io/apiserver/pkg/server"
+	genericapiserver "k8s.io/apiserver/pkg/server"
```

Comment:
This seems redundant with the prior line. Do we need an additional alias (genericapiserver) for a package we are already pulling in? We could just have server.SetupSignalHandler() below.

Reply:
yes, thx.